api/handlers.go

```diff
@@ -190,18 +190,12 @@ func (api *API) loginPOSTCredentials(w http.ResponseWriter, req *http.Request, e
 func (api *API) loginPOSTToken(w http.ResponseWriter, req *http.Request) {
 	// Fetch a JWT token from the request. This token will tell us who the user
 	// is and until when their current session is going to stay valid.
-	tokenStr, err := tokenFromRequest(req)
+	token, tokenStr, err := tokenFromRequest(req)
```

Comment:
Seems a bit redundant and prone to inconsistencies that it would return both `token` and `tokenStr` instead of just the token with a `String()` method that would be called if you ever need to represent the token as a string again.

Reply:
I was thinking about this, too. I kept the string return value because it simplifies the code elsewhere by removing the need to serialize the token and handle the error.

I removed it now and it looks OK. It still seems like an overhead to not return the string, as we already have it right there, but it's not so bad.